lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARFDebugMap.cpp

```diff
@@ -1148,16 +1175,16 @@ SymbolFileDWARFDebugMap::ParseCallEdgesInFunction(
 
 DWARFDIE SymbolFileDWARFDebugMap::FindDefinitionDIE(const DWARFDIE &die) {
   DWARFDIE result;
-  ForEachSymbolFile([&](SymbolFileDWARF *oso_dwarf) {
-    result = oso_dwarf->FindDefinitionDIE(die);
-    return result ? IterationAction::Stop : IterationAction::Continue;
-  });
+  ForEachSymbolFile(
+      "Looking up type definition", [&](SymbolFileDWARF *oso_dwarf) {
```

Comment:
Not your code, but either we should pass a reference here or check for nullptr...